tests/st/utils/docker_host.py

```diff
@@ -216,7 +217,7 @@ def start_calico_node(self, options=""):
         Start calico in a container inside a host by calling through to the
         calicoctl node command.
         """
-        args = ['node', 'run']
+        args = ['node', 'run', '--node-image=%s' % NODE_CONTAINER_NAME]
```

Comment:
I think this option should only be added if it isn't supplied in the options string.   Maybe just do a `if "--node-image" not in options`.